modules/ucfunnelBidAdapter.js

```diff
@@ -197,14 +197,38 @@ function getRequestData(bid, bidderRequest) {
     bl: language,
     je: 1,
     dnt: dnt,
-    host: host,
-    u: page,
     adid: bid.params.adid,
     tdid: userIdTdid,
     schain: supplyChain,
     fp: bid.params.bidfloor
   };
 
+  try {
+    bidData.host = window.top.location.hostname;
+    bidData.u = window.top.location.href;
+    bidData.xr = 0;
+  } catch (e) {
+    bidData.host = window.location.hostname;
+    bidData.u = document.referrer || window.location.href;
+    bidData.xr = 1;
+  }
+
+  if (window.location.ancestorOrigins && window.location.ancestorOrigins.length > 0) {
```

Comment:
hey @ucfunnel , looks like `ancestorOrigins` is not supported on IE and Firefox, given that Prebid.js aims to support these browsers (I know that it will not break since you have this condition) - would `bidderRequest.refererInfo.referer` do the job for you?

Reply:
@aleksatr It is ok with this. This parameter is for ancestorOrigins.